### ISSUE TITLE
Accessibility: ARIA widget required owned element

### ISSUE THREAD

**Issue by ravasconcelos** [ag-grid/ag-grid#8458]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Reproducible Scenario**
Accepted reproducible scenarios are
- Open in Google Chome: https://www.ag-grid.com/example-finance/
- Open the ARC Toolkit and run the tests (v5.7.2)

**Current behavior**
Multiple accessibility issues will show up, but this ticket is focused in the error below
  - "ARIA widget required owned element"
    - requiredElements: ["row"]  
    - Ex: 
```
<div class="ag-sticky-top-container" role="rowgroup" style="width: 954px; transform: translateX(0px);"></div>'
```

**Expected behavior**
No accessibility issues reported, especially the one related to "ARIA widget required owned element" due to missing row

Expected behaviour. If possible back this up with our docs/examples if possible

* **AG Grid version:** 32.0.2

* **Browser:**
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36'

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for the suggestion.

Currently there's no way to implement this behaviour with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference: **AG-12579**

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

You can follow the status of this item on the product pipeline page::
https://ag-grid.com/pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
